app/models.py

```diff
@@ -62,9 +62,9 @@ class Meta(db.Model):
   uuid = db.Column(GUID, primary_key=True, index=True, unique=True, default=lambda: uuid.uuid4())
   user_id = db.Column(GUID, db.ForeignKey('user.uuid'), nullable=False)
   note_id = db.Column(GUID, db.ForeignKey('note.uuid'), nullable=False)
-  name_encrypted = db.Column('name', db.String)
-  name_compare = db.Column(db.String)
-  kind = db.Column(db.String)
+  name_encrypted = db.Column('name', db.String(128))
```

Comment:
For all of these 128 length ones, might as well bump them to 512+ so we don't fight the length later?

Reply:
Can the name_encrypted be that long? how about 256

Reply:
512 now, as you wish